pages/ksphere/konvoy/1.5/operations/accessing-the-cluster/index.md

```diff
@@ -34,27 +36,31 @@ konvoy get ops-portal
 The URL provided in these details offers access to the **operations portal**, a web UI which links to various dashboards of the tooling integrated on Konvoy Kubernetes clusters.
 This UI is hosted on a web application within the cluster, which runs on the cluster's ingress load balancer.
 
+To access the UI:
+
 1. Enter that URL in your browser.
 2. Click the "Launch Console" button
 3. Enter your username and password as noted above, and click "Login".
 
-You will then see Konvoy's operations portal, which offers an overview of cluster status, and shortcuts to several dashboards to addon services such as Grafana.
+You will then see Konvoy's operations portal, which offers an overview of cluster status, and shortcuts to several dashboards to addon services, such as Grafana.
```

Comment:
```suggestion
You will the Konvoy's operations portal. The portal offers an overview of cluster status, and shortcuts to several dashboards to addon services, such as Grafana.
```

Reply:
I think it's more clear the other way for this one